src/modules/roc_sndio/target_pulseaudio/roc_sndio/pulseaudio_backend.cpp

```diff
@@ -57,5 +57,19 @@ ISource* PulseaudioBackend::open_source(core::IAllocator&,
     return NULL;
 }
 
+void PulseaudioBackend::get_drivers(core::Array<DriverInfo>& arr) {
+    const char* format_name = "pulseaudio";
+    size_t n;
```

Comment:
Please, do not use uninitialized variable and put its initialization inside the for-loop.